src/pkg/cli/compose/loader.go

```diff
@@ -110,10 +116,11 @@ func (c *Loader) projectOptions() (*cli.ProjectOptions, error) {
 		cli.WithDotEnv,
 
 		// DEFANG SPECIFIC OPTIONS
+		cli.WithEnv(osEnv),
```

Comment:
for implementation of environment interpolation I was looking to disable compose loader interpolation `cli.WithInterpolation(false)` so that interpolation can happen later.